### PR TITLE
KFSPTS-22312 Add Fields to Tax Transaction Details

### PR DESCRIPTION
**Pull request by ajd299**

### CODE REVIEW

File: src/main/java/edu/cornell/kfs/tax/businessobject/DvDisbursementView.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
+package edu.cornell.kfs.tax.businessobject;
+
+import org.kuali.kfs.krad.bo.TransientBusinessObjectBase;
+import org.kuali.rice.core.api.util.type.KualiInteger;
+
+/**
+ * Convenience lightweight BO to allow for ORM tools to map to the DvDisbursementView
+ */
+public class DvDisbursementView extends TransientBusinessObjectBase {
```
Comment (cah292) [inline src/main/java/edu/cornell/kfs/tax/businessobject/DvDisbursementView.java:9]: The SQL for the related view also has a column for the Payment Group ID. Should that be added to this BO as well?

If a property for that gets added here, make sure it gets included in the OJB XML file.

File: src/main/java/edu/cornell/kfs/tax/businessobject/DvDisbursementView.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    private String disbursementTypeCode;
+
+    public String getCustPaymentDocNbr() {
+        return custPaymentDocNbr;
+    }
+
+    public void setCustPaymentDocNbr(String custPaymentDocNbr) {
+        this.custPaymentDocNbr = custPaymentDocNbr;
+    }
+    public KualiInteger getDisbursementNbr() {
```
Comment (cah292) [inline src/main/java/edu/cornell/kfs/tax/businessobject/DvDisbursementView.java:23]: Missing empty line between methods.

File: src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TransactionRowDvBuilder.java [hunk 3]
```diff
@@ -332,6 +346,18 @@ void buildTransactionRows(ResultSet rs, PreparedStatement insertStatement, T sum
     }
 
 
+    private String getLedgerDocumentTypeCode(String paymentMethodCode){
```
Comment (cah292) [inline src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TransactionRowDvBuilder.java:349]: Missing space before opening curly brace.

**Review by cah292 — COMMENTED** [review]
I had a quick question about the new transient Business Object, and noticed a few minor spots where it would be good to clean up the formatting. Other than that, the code looks good.

**Comment by ajd299** [comment]
Please see my latest commits in the sql and code repositories

**Review by cah292 — APPROVED** [review]
Changes look good, unit tests pass, and the related SQL PR is also ready. Merging.

**Pull request merged**